input-events/idlharness.html

```diff
@@ -0,0 +1,40 @@
+<title>Input Event IDL tests</title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="/resources/WebIDLParser.js"></script>
+<script src="/resources/idlharness.js"></script>
+
+<pre id="untested_idl">
+interface InputEventInit {
+};
+
+interface UIEvent {
+};
+
+interface StaticRange {
+};
+</pre>
+
+<pre id='idl'>
+[Constructor(DOMString type, optional InputEventInit eventInitDict)]
+interface InputEvent : UIEvent {
```

Comment:
Sure, I'd love for you to write some tests in Chromium for exporting, so we can uncover ever more bugs before a full launch.

I like squashed commits, but if I'm the one merging a PR I'll just use the "Squash and merge" button and tweak the message to my liking, so it doesn't matter much.